plugins/policy-recommendation/antrea_crd.py

```diff
@@ -0,0 +1,627 @@
+# This file could be changed to auto-generated by using openAPI generator in the future like the K8s python lib.
+
+class NetworkPolicy(object):
+    attribute_types = {
+        "kind": "string",
+        "api_version": "string",
+        "metadata": "kubernetes.client.V1ObjectMeta",
+        "spec": "NetworkPolicySpec",
+        "status": "NetworkPolicyStatus"
+    }
+
+    def __init__(self, kind=None, api_version=None, metadata=None, spec=None, status=None):
+        self.kind = kind
+        self.api_version = api_version
+        self.metadata = metadata
+        self.spec = spec
+        self.status = status
+
+    def to_dict(self):
+        """Returns the model properties as a dict"""
+        result = {}
+
+        for attr, _ in self.attribute_types.items():
+            value = getattr(self, attr)
+            if isinstance(value, list):
+                result[attr] = list(map(
+                    lambda x: x.to_dict() if hasattr(x, "to_dict") else x,
+                    value
+                ))
+            elif hasattr(value, "to_dict"):
+                result[attr] = value.to_dict()
+            elif isinstance(value, dict):
+                result[attr] = dict(map(
+                    lambda item: (item[0], item[1].to_dict())
+                    if hasattr(item[1], "to_dict") else item,
+                    value.items()
+                ))
+            else:
+                result[attr] = value
+
+        return result
+
+    
+class NetworkPolicySpec(object):
+    attribute_types = {
+        "tier": "string",
+        "priority": "float",
+        "applied_to": "list[NetworkPolicyPeer]",
+        "ingress": "list[Rule]",
+        "egress": "list[Rule]"
+    }
+
+    def __init__(self, tier=None, priority=None, applied_to=None, ingress=None, egress=None):
+        self.tier = tier
+        self.priority = priority
+        self.applied_to = applied_to
+        self.ingress = ingress
+        self.egress = egress
+    
+    def to_dict(self):
+        """Returns the model properties as a dict"""
+        result = {}
+
+        for attr, _ in self.attribute_types.items():
+            value = getattr(self, attr)
+            if isinstance(value, list):
+                result[attr] = list(map(
+                    lambda x: x.to_dict() if hasattr(x, "to_dict") else x,
+                    value
+                ))
+            elif hasattr(value, "to_dict"):
+                result[attr] = value.to_dict()
+            elif isinstance(value, dict):
+                result[attr] = dict(map(
+                    lambda item: (item[0], item[1].to_dict())
+                    if hasattr(item[1], "to_dict") else item,
+                    value.items()
+                ))
+            else:
+                result[attr] = value
+
+        return result
+
+class NetworkPolicyPeer(object):
+    attribute_types = {
+      "ip_block": "IPBlock",
+      "pod_selector": "kubernete.client.V1LabelSelector",
```

Comment:
nit: kubernete->kubernetes here and below

Reply:
Thanks, addressed.